_includes/content/ctl-kubectl-plugin-install.md

```diff
@@ -9,20 +9,7 @@ you want to install the binary.
 
 1. Use the following command to download the `calicoctl` binary.
 
-   ```bash
-   curl -o kubectl-calico -L  {% include urls component="calicoctl" %}
-   ```
-
-1. Set the file to be executable.
-
-   ```bash
-   chmod +x kubectl-calico
-   ```
-
-   > **Note**: If the location of `kubectl-calico` is not already in your `PATH`, move the file
-   > to one that is or add its location to your `PATH`. This is required in order for
-   > kubectl to detect the plugin and allow you to use it.
-   {: .alert .alert-info}
+   {% include urls component="calicoctl" plugin="true" %}
```

Comment:
If this include is going to have more than just the URL in it now, then we should change what it is called.

e.g., include calicoctl-tabs

or something like that.